Add docker information (#5)
liteflow-labs/docs: README.md

npm install -g @liteflow/cli
```

> Not that Liteflow also required Docker for local development so make sure to [install and run Docker](https://docs.docker.com/get-docker/).

Reviewer: ```suggestion
Liteflow requires Docker for local development so make sure to [install and start Docker](https://docs.docker.com/get-docker/).
```
